src/Flex.php

```diff
@@ -48,7 +48,7 @@ class Flex implements PluginInterface, EventSubscriberInterface
     private $originalLockHash;
     private static $activated = true;
 
-    public function activate(Composer $composer, IOInterface $io)
+    public function activate(Composer $composer, IOInterface $io): void
```

Comment:
I'm not sure about this one. The `activate()` method is part of a Composer built-in interface, in which the method return type is not defined.

Shall I revert this change?